src/Emogrifier.php

```diff
@@ -375,7 +375,10 @@ protected function process(\DOMDocument $xmlDocument)
             $allCss .= $this->getCssFromAllStyleNodes($xPath);
         }
 
-        $cssParts = $this->splitCssAndMediaQuery($allCss);
+        $splitCss = $this->splitCssAndMediaQuery($allCss);
+        // this is temporary pending refactoring `parseCssRules` and `copyCssWithMediaToStyleNode`
```

Comment:
I don't think we need this comment anymore. Even if we stopped after this PR, the refactoring would make sense.

Reply:
Removed